.circleci/config.yml

```diff
@@ -84,20 +84,18 @@ jobs:
     environment:
       MAGENTO_DIR: magento
     steps:
+      - checkout
       - run: |
-          .circleci/scripts/rc-tag.sh
-      - when:
-          condition: $TAGTHISWEEK
-          steps:
-            - run:
-                command: |
-                  apt-get update
-                  apt-get install -y git
-                  apt-get install -y openssh-server
-            - swissknife/publish_github_release:
-                tag: $NEWTAGNAME
-            - notify-rc-tag:
-                tag: $NEWTAGNAME
+          source .circleci/scripts/rc-tag.sh
+          echo "export NEWTAGNAME=$NEWTAGNAME" >> $BASH_ENV
+      - run: |
+          sudo apt-get update
+          sudo apt-get install -y git
+          sudo apt-get install -y openssh-server
+      - swissknife/publish_github_release:
```

Comment:
We can one more problem here. When we call this command it creates commit, not tag.
Commis doesn't have its `creatordate`. Instead, it has the same creatordate as the master branch has.
So we can't rely on any date to count if three weeks passed or not.

@roopakv Can we create tag instead of commit? Or we need to find other way how to figured out if three weeks passed or not?

CC: @matt-thomason